h2o-py/h2o/h2o.py

```diff
@@ -86,7 +86,7 @@ def connect(server=None, url=None, ip=None, port=None, https=None, verify_ssl_ce
                                      auth=auth, verify_ssl_certificates=verify_ssl_certificates,
                                      proxy=proxy,cookies=cookies,
                                      verbose=verbose)
-        h2oconn.cluster.timezone = "UTC"
+        # h2oconn.cluster.timezone = "UTC"  It's now set to UTC by default
```

Comment:
I don't think the comment is necessary. Can we include the cluster time zone in the cluster status?

```
    H2O cluster uptime:         5 seconds 679 milliseconds 
    H2O cluster version:        3.17.0.99999 
    H2O cluster version age:    1 day  
    H2O cluster name:           mkurka 
    H2O cluster total nodes:    1 
    H2O cluster total memory:   3.56 GB 
    H2O cluster total cores:    8 
    H2O cluster allowed cores:  8 
    H2O cluster healthy:        TRUE 
    H2O Connection ip:          localhost 
    H2O Connection port:        54321 
    H2O Connection proxy:       NA 
    H2O Internal Security:      FALSE 
    H2O API Extensions:         AutoML, XGBoost, Algos, Core V3, Core V4 
    R Version:                  R version 3.3.1 (2016-06-21) 
```

Reply:
I like the comment, to explain the change, but it's not necessary.

There are two timezones: one in DateTimeZone for the cluster itself and used for logging, and one for datafile parsing.  I suppose we could add those, but to me it seems overkill.